central/policy/service/validator_test.go

```diff
@@ -168,12 +168,12 @@ func (s *PolicyValidatorTestSuite) TestValidateDescription() {
 	policy = &storage.Policy{
 		Description: `This policy is the stop when an image is terrible and will cause us to lose lots-o-dough. Why? Cause Money!
 			Oh, and I almost forgot that this is also to help the good people of nowhere-ville get back on their
-			feet after that tornado ripped their town to shreds and left them nothing but pineapple and gum.  It was the It was 
-			the best of times, it was the worst of times, it was the age of wisdom, it was the age of foolishness, it was the 
-			epoch of belief, it was the epoch of incredulity, it was the season of Light, it was the season of Darkness, it was 
-			the spring of hope, it was the winter of despair, we had everything before us, we had nothing before us, we were all 
-			going direct to Heaven, we were all going direct the other way--in short, the period was so far like the present 
-			period that some of its noisiest authorities insisted on its being received, for good or for evil, in the superlative 
+			feet after that tornado ripped their town to shreds and left them nothing but pineapple and gum.  It was the It was
+			the best of times, it was the worst of times, it was the age of wisdom, it was the age of foolishness, it was the
+			epoch of belief, it was the epoch of incredulity, it was the season of Light, it was the season of Darkness, it was
+			the spring of hope, it was the winter of despair, we had everything before us, we had nothing before us, we were all
+			going direct to Heaven, we were all going direct the other way--in short, the period was so far like the present
+			period that some of its noisiest authorities insisted on its being received, for good or for evil, in the superlative
```

Comment:
Just checking if you change this on purpose.

Reply:
I don't think it matters. Judging by the code the test checks if the `Policy Description` has more than 800 chars.